compiler/rustc_passes/src/liveness.rs

```diff
@@ -1521,6 +1521,12 @@ impl<'tcx> Liveness<'_, 'tcx> {
     }
 
     fn warn_about_unused_args(&self, body: &hir::Body<'_>, entry_ln: LiveNode) {
+        if let Some(intrinsic) = self.ir.tcx.intrinsic(body.value.hir_id.owner.def_id) {
```

Comment:
```suggestion
        if let Some(intrinsic) = self.ir.tcx.intrinsic(self.ir.tcx.hir().body_owner_def_id(body){
```

or something similar to this should also work